src/libraries/System.Security.Cryptography/tests/ShakeTestDriver.cs

```diff
@@ -12,7 +12,7 @@
 
 namespace System.Security.Cryptography.Tests
 {
-    public interface IShakeTrait<TShake> where TShake : IDisposable, new()
+    public interface IShakeTrait<TShake> where TShake : class, IDisposable, new()
```

Comment:
New constraint by `class` is to make `Assert.{Not}Same` happy that the type is a reference type.